caravel/assets/javascripts/explorev2/components/TimeFilter.jsx

```diff
@@ -45,13 +46,13 @@ const TimeFilter = (props) => {
     {
       key: 'since',
       title: 'Since',
-      options: sinceOptions.map((s) => ({ value: s, label: s })),
+      options: getSelectOptions(sinceOptions),
```

Comment:
i wonder if we should call this `formatSelectOptions` since it's returning a new format of `{ value: s, label: s}`